src/Neo/SmartContract/Native/PolicyContract.cs

```diff
@@ -588,11 +598,16 @@ private bool UnblockAccount(ApplicationEngine engine, UInt160 account)
         {
             AssertCommittee(engine);
 
-
             var key = CreateStorageKey(Prefix_BlockedAccount, account);
             if (!engine.SnapshotCache.Contains(key)) return false;
 
             engine.SnapshotCache.Delete(key);
+
+            // Remove request funds if any
+
+            key = CreateStorageKey(Prefix_BlockedAccountRequestFunds, account);
+            engine.SnapshotCache.Delete(key);
```

Comment:
Does it need an event?

Reply:
https://github.com/neo-project/neo/issues/4387